src/ast/mod.rs

```diff
@@ -551,12 +551,34 @@ pub enum Statement {
     Rollback { chain: bool },
     /// CREATE SCHEMA
     CreateSchema { schema_name: ObjectName },
-
     /// ASSERT <condition> [AS <message>]
     Assert {
         condition: Expr,
         message: Option<Expr>,
     },
+    /// DEALLOCATE [ PREPARE ] { name | ALL }
+    ///
+    /// Note: this is a PostgreSQL-specific statement.
+    Deallocate {
+        name: Option<ObjectName>,
+        all: bool,
+        prepare: bool,
```

Comment:
This seems to be meaningless syntax. It's fine to keep it, but maybe note that PostgreSQL docs say it's ignored?

Reply:
Yes. The syntax is meaningless. But I think if we should output (format) the useless keyword `PREPARE` as the SQL passed into. What do you think?